client/src/pages/Login/useStyles.ts

```diff
@@ -1,21 +1,75 @@
 import { makeStyles } from '@material-ui/core/styles';
 
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles((theme) => ({
```

Comment:
Could you add a type to the theme argument. `theme: Theme`

Reply:
Changes amended in new PR push.